go/README.md

```diff
@@ -37,7 +37,7 @@ go run server.go
 Some European institutions require an OAuth redirect authentication flow, where the end user is redirected to the bank’s website or mobile app to authenticate. For this flow, you should provide two additional configuration parameters, `PLAID_OAUTH_NONCE` and `PLAID_OAUTH_REDIRECT_URI`.
 
 ``` bash
-# You will need to whitelist the PLAID_OAUTH_REDIRECT_URI for
+# You will need to allow the PLAID_OAUTH_REDIRECT_URI for
```

Comment:
I suggest using `configure` instead of `allow` for consistency with our [docs](https://plaid.com/docs/#oauth).

```suggestion
# You will need to configure the PLAID_OAUTH_REDIRECT_URI for
```